### PR TITLE
 docs(PPDSC-2667) image documentation

### PR DESCRIPTION
**Pull request by RashikaNewsUK**
PPDSC-2667

**What**

1. Background - why this is needed
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [x] Updated relevant documentation

**I have tested manually:**
- [x] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [x] The screen reader reads and flows through the elements as expected.
- [x] There are no new errors in the browser console coming from this PR.
- [x] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [x] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2667-image-documentation
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2667-image-documentation/storybook

**Review by mstuartf — APPROVED** [review]

File: site/pages/components/image.tsx [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+const infoIcon = (
+  <IconFilledInfo
+    overrides={{
+      size: 'iconSize020',
+    }}
+  />
+);
+
+const ImageComponent = (layoutProps: LayoutProps) => (
+  <ComponentPageTemplate
```
Comment (mutebg) [inline site/pages/components/image.tsx:25]: You can check how we started typing this template for example in Slider page.
```suggestion
  <ComponentPageTemplate<typeof Image>
```

File: site/pages/components/image.tsx [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+              {
+                label: 'imageCircle',
+                value: {
+                  stylePreset: 'imageCircle',
+                },
+              },
+            ],
+          },
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        ] as any,
```
Comment (mutebg) [inline site/pages/components/image.tsx:133]: Once you type the page, you can remove this `any`

File: site/pages/components/image.tsx [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+              type: 'Override<NewsKitIcon>',
+              description:
+                'If provided, this overrides the default placeholder icon.',
+            },
+            {
+              attribute: 'logicalProps',
+              type: 'MQ<string>',
+              description:
+                'Can take one space token to specify the logical inline start and end padding of the container. Can be used on breakpoints.',
+            },
```
Comment (mutebg) [inline site/pages/components/image.tsx:504]: This is not correct, can you please check how we do that in Sluder you need to do it like `...logicalMarginOverrideProps`

**Review by mutebg — CHANGES_REQUESTED** [review]

File: site/pages/components/image.tsx (hunk 1 above)
Comment (RashikaNewsUK) [reply site/pages/components/image.tsx:25]: @mutebg will check -thanks!

File: site/pages/components/image.tsx (hunk 2 above)
Comment (RashikaNewsUK) [reply site/pages/components/image.tsx:133]: @mutebg I have added this` any` to accept all the type of props. Is that need to change anything?

File: site/pages/components/image.tsx (hunk 3 above)
Comment (RashikaNewsUK) [reply site/pages/components/image.tsx:504]: Will check and modify, -thanks!

File: site/pages/components/image.tsx [hunk 4]
```diff
@@ -74,16 +78,16 @@ const ImageComponent = (layoutProps: LayoutProps) => (
...
-            name: 'max height',
-            propName: 'maxHeight',
-            value: '200',
-          },
-          {
-            name: 'max width',
-            propName: 'maxWidth',
-            value: '200',
-          },
+          // {
```
Comment (mutebg) [inline site/pages/components/image.tsx:81]: Remove this comment if you don't need it

**Review by mutebg — APPROVED** [review]

**Review by mstuartf — APPROVED** [review]

**Pull request merged**